website/client/src/components/settings/subscription.vue

```diff
@@ -1,5 +1,23 @@
 <template>
   <div class="standard-page pt-0 px-0">
+    <div class="banner-g1g1 mx-n3 d-flex justify-content-center">
```

Comment:
@tsukimi2 did you add this code? It might be the result of some merging gone wrong because I don't think it's supposed to be here

Reply:
@paglias Yes, I think it's merging gone wrong, I didn't add this code.  It's been deleted in the new push.